config/config.go

```diff
@@ -22,7 +22,7 @@ const (
 	// SettingListen is the config key for the listen address
 	SettingListen = "listen"
 	// SettingListenDefault is the default value for the listen address
-	SettingListenDefault = ":8080"
+	SettingListenDefault = "workflows:8080"
```

Comment:
Why?

Reply:
Oops.. This is just a goof from when I was playing around. I didn't actually notice as the default `config.yaml` file overwrites it.